### ISSUE TITLE
Uncaught TypeError: Cannot read property 'endsWith' of undefined

### ISSUE THREAD

**Issue by AguirreDavid** [gangstead/atom-scala-format#1]
Uncaught TypeError: Cannot read property 'endsWith' of undefined

**Atom Version**: 0.162.0
**System**: Mac OS X 10.10.1
**Thrown From**: [scala-format](https://github.com/gangstead/atom-scala-format) package, v0.1.2
### Steps To Reproduce
1. ...
2. ...
### Stack Trace

```
At /Users/fenix/.atom/packages/scala-format/lib/scala-format.js:61

TypeError: Cannot read property 'endsWith' of undefined
  at Object.module.exports.format (/Users/fenix/.atom/packages/scala-format/lib/scala-format.js:61:17)
  at atom-workspace.<anonymous> (/Users/fenix/.atom/packages/scala-format/lib/scala-format.js:23:22)
  at atom-workspace.handler (/Applications/Atom-2.app/Contents/Resources/app/src/space-pen-extensions.js:115:32)
  at atom-workspace.jQuery.event.dispatch (/Applications/Atom-2.app/Contents/Resources/app/node_modules/space-pen/vendor/jquery.js:4681:9)
  at atom-workspace.elemData.handle (/Applications/Atom-2.app/Contents/Resources/app/node_modules/space-pen/vendor/jquery.js:4359:46)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom-2.app/Contents/Resources/app/src/command-registry.js:243:29)
  at CommandRegistry.handleCommandEvent (/Applications/Atom-2.app/Contents/Resources/app/src/command-registry.js:3:61)
  at CommandRegistry.module.exports.CommandRegistry.dispatch (/Applications/Atom-2.app/Contents/Resources/app/src/command-registry.js:156:19)
  at Ipc.<anonymous> (/Applications/Atom-2.app/Contents/Resources/app/src/window-event-handler.js:65:30)
  at Ipc.emit (events.js:107:17)
  at process.<anonymous> (/Applications/Atom-2.app/Contents/Resources/atom/renderer/api/lib/ipc.js:22:29)
  at process.emit (events.js:110:17)

```

/cc @atom/core

**Comment by gangstead** [comment]
Thanks for the bug report. Is it possible to post the scala code you are trying to format?

**Comment by gangstead** [comment]
Testing a fix right now.

**Comment by gangstead** [comment]
fixed in 0.1.3 with Atom 0.181 or above.
